Отправлять сообщения в лог по нажатию кнопки или по нажатию клавиши E…

njakimov opened the pull request:
…nter.

Создать лог в файле (показать комментарием, где и как Вы планируете писать сообщение в файловый журнал).

Commit:
Отправлять сообщения в лог по нажатию кнопки или по нажатию клавиши Enter.
Создать лог в файле (показать комментарием, где и как Вы планируете писать сообщение в файловый журнал).

Ivan-Igorevich commented on src/chat/lesson4/online/ClientGUI.java:
```diff
@@ -0,0 +1,160 @@
+        add(panelBottom, BorderLayout.SOUTH);
+
+        btnSend.addActionListener(this);
+        tfMessage.addKeyListener(new KeyAdapter() {
```
там у текстФилда есть ЭкшнЛистнер, так вот он (очень удачно) срабатывает как раз в момент, когда пользователь нажимает на энтер, находясь в активном текстФилде

Commit:
Отправлять сообщения в лог по нажатию кнопки или по нажатию клавиши Enter.
Создать лог в файле (показать комментарием, где и как Вы планируете писать сообщение в файловый журнал).
